poppy/fresnel.py

```diff
@@ -39,19 +40,21 @@ class QuadPhase(poppy.optics.AnalyticOpticalElement):
 
     """
 
+    @utils.quantity_input(z=u.m)
     def __init__(self,
-                 z,  # FIXME consider renaming fl? z seems ambiguous with distance.
+                 z=1*u.m,  # FIXME consider renaming fl? z seems ambiguous with distance.
                  planetype=PlaneType.intermediate,
                  name='Quadratic Wavefront Curvature Operator',
                  **kwargs):
         poppy.AnalyticOpticalElement.__init__(self, name=name, planetype=planetype, **kwargs)
         self.z = z
 
-        if isinstance(z, u.quantity.Quantity):
-            self.z_m = z.to(u.m)  # convert to meters.
-        else:
-            _log.debug("Assuming meters, phase {:.3g} has no units for Optic: " .format(z) + self.name)
-            self.z_m = z * u.m
+        #if isinstance(z, u.quantity.Quantity):
```

Comment:
Although, ha, actually this whole part of the code can be simplified dramatically since before we were carrying both a `self.z` which was a float and a  `self.z_m` which was a Quantity in meters.  There's probably no longer any reason to do that.